### PR TITLE
Allow kwargs in Wandb & Neptune + kwargs docstring

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Add kwargs in Wandb to allow any additional parameter that might be added to wandb in future without any change. 

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/loggers/wandb.py [hunk 1]
```diff
@@ -51,11 +51,8 @@ class WandbLogger(LightningLoggerBase):
         anonymous: Enables or explicitly disables anonymous logging.
         version: Sets the version, mainly used to resume a previous run.
         project: The name of the project to which this run will belong.
-        tags: Tags associated with this run.
         log_model: Save checkpoints in wandb dir to upload on W&B servers.
         experiment: WandB experiment object
-        entity: The team posting this run (default: your username or your default team)
-        group: A unique string shared by all runs in a given group
```
Comment (SkafteNicki) [inline pytorch_lightning/loggers/wandb.py:58]: We should probably describe that additional keywords (mention group, tag ect by name) used by wanddb init can be feed in as keywords when initializing the logger

Comment (rohitgr7) [reply pytorch_lightning/loggers/wandb.py:58]: is it good now??

Comment (SkafteNicki) [reply pytorch_lightning/loggers/wandb.py:58]: Very good :]

File: pytorch_lightning/loggers/wandb.py [hunk 2]
```diff
@@ -51,11 +51,10 @@ class WandbLogger(LightningLoggerBase):
         anonymous: Enables or explicitly disables anonymous logging.
         version: Sets the version, mainly used to resume a previous run.
         project: The name of the project to which this run will belong.
-        tags: Tags associated with this run.
         log_model: Save checkpoints in wandb dir to upload on W&B servers.
-        experiment: WandB experiment object
-        entity: The team posting this run (default: your username or your default team)
-        group: A unique string shared by all runs in a given group
+        experiment: WandB experiment object.
+        kwargs: Addition arguments like `entity`, `group`, `tags`, etc. used by
```
Comment (awaelchli) [inline pytorch_lightning/loggers/wandb.py:56]:
```suggestion
        kwargs: Additional arguments like `entity`, `group`, `tags`, etc. used by
```

**Review by awaelchli — COMMENTED** [review]
do we need to do it for other loggers too?

**Comment by rohitgr7** [comment]
> do we need to do it for other loggers too?

for others `kwargs` is already there but docstring is not there for it. I'll add it sometime later.

**Review by awaelchli — CHANGES_REQUESTED** [review]
@rohitgr7 with my comment before, I didn't mean to say you need to do all in one PR. 
please only do it for wandb here, and for each other logger a separate PR. 
we need to keep these PRs short and easy to review.

**Comment by rohitgr7** [comment]
@awaelchli just added the docstring for `kwargs` here wherever it was missing for other loggers and removed few parameters that can be passed as `kwargs`. The `kwargs` parameter was already present for those loggers except for wandb. Other changes are just isort, so nothing much is changed here.

**Review by Borda — APPROVED** [review]
LGTM 👍

**Review by awaelchli — APPROVED** [review]

**Comment by Borda** [comment]
@rohitgr7 mind rebase :]

**Review by SkafteNicki — APPROVED** [review]
LGTM :]

**Pull request merged**